frontend/src/components/form/hash-input.jsx

```diff
@@ -0,0 +1,59 @@
+import { createEffect, createSignal } from "solid-js";
+import { styled } from "solid-styled-components";
+import icon from "../../assets/copy-ico.svg";
+
+const HashInput = (props) => {
+  const value = () => props.value || "";
+  const [hasValue, setHasValue] = createSignal(Boolean(props.defaultValue));
```

Comment:
Do we really need this flag as a separate signal?

Reply:
removed in result of refactoring with solid-forms